src/ServiceControl.Monitoring/Metrics/Raw/LongValueOccurenceSerializerDefinition.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace ServiceControl.Monitoring.Processing.RawData
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using global::NServiceBus.MessageInterfaces;
+    using global::NServiceBus.Serialization;
+    using global::NServiceBus.Settings;
+    using NServiceBus.Metrics;
+
+    class LongValueOccurrenceSerializerDefinition : SerializationDefinition
```

Comment:
I agree that performance is not enough with current approach. That being said the structure of data stored in memory will very soon change to include longer historical intervals and data for purposes of details page. 
As a result the plan is to wait until we have better knowledge about what we want to store and tackle performance at that point.